modules/custom/activity_viewer/src/Plugin/views/row/ActivityRow.php

```diff
@@ -30,8 +31,32 @@ public function preRender($result) {
 
       foreach ($result as $row) {
         $render_result = [];
-        $render_result[] = $row;
         $entity = $row->_entity;
+        $target_type = $entity->get('field_activity_entity')->getValue()[0]["target_type"];
```

Comment:
There are better Drupal functions to use to get a fields value, also make sure you make this a bit more defensive what if this is empty for some reason? :)